stdlib/http/src/main/ballerina/src/http/websocket/websocket_client.bal

```diff
@@ -201,7 +223,7 @@ public type WebSocketClientConfiguration record {|
     int maxFrameSize = 0;
     boolean webSocketCompressionEnabled = true;
     int handShakeTimeoutInSeconds = 300;
-    WebSocketRetryConfig retryConfig?;
+    map<string> cookies?;
```

Comment:
Shall we make the cookies  an array of Cookie rather than having it as a  `map<string>`

Reply:
Fixed in  #24770